.pre-commit-config.yaml

```
@@ -84,12 +79,11 @@ repos:
     hooks:
       - id: black
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.172"
     hooks:
-      - id: flake8
-        args:
-          - --count
+      - id: ruff
+        args: ["--fix"]
```

```suggestion
        args: ["--fix", "--force-exclude"]
```

Add the `--force-exclude` flag.